src/App.js

```diff
@@ -38,6 +39,10 @@ const App = () => {
           <Drawer.Screen name="Wait" component={WaitScreen} />
           <Drawer.Screen name="Date Picker" component={DatePickerScreen} />
           <Drawer.Screen name="Scroll View" component={ScrollViewScreen} />
+          <Drawer.Screen
```

Comment:
why it is not in a single line like all the others?

Reply:
it was formatted by linter due to line length